rsc/src/test/java/com/cloudera/livy/rsc/rpc/TestRpc.java

```diff
@@ -186,6 +188,25 @@ public void testEncryption() throws Exception {
     assertEquals(outbound.message, reply.message);
   }
 
+  @Test
+  public void testPortRange() throws Exception {
+    String portRangeData = emptyConfig.get(LAUNCHER_PORT_RANGE);
+    String portRange[] = portRangeData.split("~");
+    int startPort = Integer.parseInt(portRange[0]);
+    int endPort = Integer.parseInt(portRange[1]);
+    for (int index = startPort; index <= endPort; index++) {
+      RpcServer server = autoClose(new RpcServer(emptyConfig));
+      assertTrue(startPort <= server.getPort() && server.getPort() <= endPort);
```

Comment:
Seems you are using the default value of LAUNCHER_PORT_RANGE, don't understand why using a for loop here ?

Basically I think we need to add 2 test cases. One is specifying a wrong value of LAUNCHER_PORT_RANGE and verify the correct Exception is caught, another is specifying a correct value of LAUNCHER_PORT_RANGE, and the port of server is in this range.